employees/tests/test_unit_report_model.py

```diff
@@ -155,6 +162,22 @@ def test_report_model_work_hours_str_property_should_return_work_hours_field_val
         self.assertEqual(report.work_hours_str, "8:00")
 
 
+class TestReportWorkHoursSumForGivenDayForSingleUser(TestCase):
```

Comment:
maybe add one test for maximal allowed value (24)?

Reply:
Nice idea, added :+1: